### DISCUSSION TITLE
Is there a way to "free" the range selection on start or end?

### DISCUSSION THREAD

**Discussion by megasmack** [easepick/easepick#190] [Q&A]
Is there a method or event I can use to "free" the range selection for the start and/or end date? I realize this is what the `repick` option is for, but for a variety of reasons I need to disable `autoApply` and enable `inline`. Basically trying to recreate the repick function, but using custom javascript events instead.

**Comment by idbrian** [comment]
I'm just trying to do the same thing, and I still can't do it. did you manage to do it?

**Reply by megasmack** [reply]
I wasn't. I ended up reverting to a more traditional use of easepick where `inline` was false.
